### DISCUSSION TITLE
【Dropdown组件】自定义 dropdownRender 问题

### DISCUSSION THREAD

**Discussion by consistent-k** [ant-design/ant-design#44077] [Q&A]
今天发现低于Chrome87版本时 dropdownRender展示不出来。看Dom节点上会加载样式 left: -1000vw; top: -1000vh;
然后找了源码，发现单测跑出来的__snapshots__文件内也是如此，求指点这里的原因和兼容方案。
![image](https://github.com/ant-design/ant-design/assets/67736912/be5fcb68-4411-4a88-835a-4b8cdc78c645)

**Comment by afc163** [comment]
看下 https://ant.design/docs/react/compatible-style-cn

**Reply by consistent-k** [reply]
嗯 这个我看到了 目前已经是传的hashPriority="high" 渲染出来的组件确实没有 inset了 但是top 和 left的值计算出来为什么是-1000 这个我不是很明白。

**Reply by consistent-k** [reply]
求助 是不是因为 @ant-design/cssinjs 在做兼容的时候是运行时判断，但是Dropdown组件的样式也是动态计算的，这两个的先后顺序导致的-1000呀
